js-sandbox/src/script.rs

```diff
@@ -102,18 +103,19 @@ impl Script {
 		Ok(result)
 	}
 
+	/// Bind this script to an API
```

Comment:
Added missing trait doc

Reply:
see above

Reply:
Same as above; it's just a placeholder so the ci passed, not sure what the method does